src/ExceptionReporter.Shared/ReportGenerator.cs

```diff
@@ -46,8 +45,10 @@ public ReportGenerator(ExceptionReportInfo reportInfo)
 
 		private string GetAppVersion()
 		{
-			return ApplicationDeployment.IsNetworkDeployed ? 
-				ApplicationDeployment.CurrentDeployment.CurrentVersion.ToString() : _info.AppAssembly.GetName().Version.ToString();
+			return _info.AppAssembly.GetName().Version.ToString();
+			// lost during migration to net5.0
```

Comment:
That's a bugger isn't it...
On my current team, this ClickOnce deployed version is quite important - we might have to use conditional code/preprocessing or something to keep this...